src/server/server_family.cc

```diff
@@ -1976,6 +2057,41 @@ void ServerFamily::ReplicaOf(CmdArgList args, ConnectionContext* cntx) {
   }
 }
 
+void ServerFamily::ReplicaOf(CmdArgList args, ConnectionContext* cntx) {
+  ReplicaOfInternal(args, cntx, true, true);
+}
+
+void ServerFamily::Replicate(string& host, string& port, bool* success) {
```

Comment:
Any reason not to return bool as success instead of having an out param?

Reply:
No particular reason :smile: